README.md

```diff
@@ -4,6 +4,14 @@ Pipeline for ingesting nationwide feed of vaccine facilities
 
 ## Usage
 
+### Quick Setup For MacOS Homebrew users
```

Comment:
I tried to make the steps environment agnostic and I test them on my mac and linux box. What issues did you run into when you tried them?

The steps you proposed aren't quite right because they could install the wrong version of python, and they suggest the user change their system python to run our project which many users will be loath to do if they do other python work.

Reply:
If you are trying to avoid `pyenv` install then we could add that as an optional by suggested step?